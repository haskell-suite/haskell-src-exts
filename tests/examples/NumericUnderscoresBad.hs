-- Missing NumericUnderscores extension. Should fail.
f :: Integer -> ()
f 1_000 = ()
f _   = ()
