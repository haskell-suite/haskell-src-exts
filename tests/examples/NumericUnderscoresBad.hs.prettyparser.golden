ParseFailed (SrcLoc "tests/examples/NumericUnderscoresBad.hs" 3 1) "arity mismatch for 'f'"
